provider/pkg/provider/provider.go

```diff
@@ -460,6 +460,10 @@ func (k *googleCloudProvider) Update(ctx context.Context, req *rpc.UpdateRequest
 	}
 
 	uri := res.RelativePath(req.GetId())
+	if strings.HasSuffix(uri, ":getIamPolicy") {
```

Comment:
Would it make sense to set a separate `UpdatePath` field in the metadata to encode exceptions such as these where the IdPath can't be used for updates?

Reply:
I started with that but I thought it was a bit over-engineered for the single use case that we have right now. WDYT?

Reply:
Understood. Cut a ticket so we can revisit?

Reply:
https://github.com/pulumi/pulumi-gcp-native/issues/42